src/Engine/include/OpenLoco/Engine/Input/ShortcutManager.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <vector>
+
+namespace OpenLoco::Input
+{
+    enum class Shortcut : uint32_t;
+}
+
+namespace OpenLoco::Input::ShortcutManager
+{
+    using string_id = uint16_t;
+
+    struct KeyboardShortcut
```

Comment:
Should we come up with a new name for this struct if its no longer just for keyboards?

Reply:
I renamed it to ShortcutEntry for the time being.